wiring/src/spark_wiring_fuel.cpp

```diff
@@ -117,14 +127,24 @@ float FuelGauge::getVCell() {
 
 // Read and return the state of charge of the cell
 float FuelGauge::getSoC() {
-
     byte MSB = 0;
     byte LSB = 0;
 
-    if(readRegister(SOC_REGISTER, MSB, LSB) != SYSTEM_ERROR_NONE) {
+    if (readRegister(SOC_REGISTER, MSB, LSB) != SYSTEM_ERROR_NONE) {
         return -1.0f;
     }
-    return particle::detail::_getSoC(MSB, LSB);
+
+    // Load SoC Bit Precision from SystemPowerConfiguration
+    int soc_bits = particle::power::DEFAULT_SOC_18_BIT_PRECISION;
+#if (PLATFORM_ID != PLATFORM_GCC)
```

Comment:
`HAL_PLATFORM_POWER_MANAGEMENT`. If this is affecting unit tests, might make sense to add `system_power_management_get_config()` stub.